ops/services/web_application_firewall/main.tf

```diff
@@ -72,11 +72,10 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
       version = "3.2"
 
       rule_group_override {
-        rule_group_name = "REQUEST-942-APPLICATION-ATTACK-SQLI"
+        rule_group_name = "REQUEST-920-PROTOCOL-ENFORCEMENT" //TODO: add exception for whoami
         disabled_rules = [
-          "942430",
-          "942260",
-          "942200"
+          "920300",
```

Comment:
I don't see this rule id in the [Azure docs](https://docs.microsoft.com/en-us/azure/web-application-firewall/ag/application-gateway-crs-rulegroups-rules?tabs=owasp32#crs920-32), am I missing something?

Reply:
This is a great catch! Unfortunately, Azure does not have this code covered in their documentation. It is repeatedly listed in our firewall logs, though, and it corresponds to "Request Missing an Accept Header".